### PR TITLE
feat: Connect alarm data from server and display

### LINKED ISSUE #50: feat: Connect alarm data from server and display

### PR DESCRIPTION
**Pull request by jonlld**

### CODE REVIEW

File: src/components/molecules/AlarmHistoryItem.tsx [hunk 1]
```diff
@@ -3,18 +3,20 @@ import styles from "@/styles/components/molecules/AlarmHistoryItem.module.scss";
 
 type Props = {
   data: {
+    is_alarm: boolean;
```
Comment (Takashicc) [inline src/components/molecules/AlarmHistoryItem.tsx]: Are you using this `is_alarm`?

File: src/components/organisms/AlarmHistory.tsx [hunk 2]
```diff
@@ -1,129 +1,151 @@
...
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+//   {
+//     date: "01/01/22",
+//     time: "14:22",
+//   },
+// ];
```
Comment (Takashicc) [inline src/components/organisms/AlarmHistory.tsx]: Delete this comments.

File: src/components/organisms/AlarmHistory.tsx [hunk 3]
```diff
@@ -1,129 +1,151 @@
...
+
+type AlarmData = {
+  is_alarm: boolean;
```
Comment (Takashicc) [inline src/components/organisms/AlarmHistory.tsx]: Is `is_alarm` used?

**Review by Takashicc — CHANGES_REQUESTED** [review]

File: src/components/molecules/AlarmHistoryItem.tsx (hunk 1 above)
Comment (jonlld) [reply src/components/molecules/AlarmHistoryItem.tsx:6]: It's not used but it is being sent from the server, so probably need to include for TS?

File: src/components/organisms/AlarmHistory.tsx (hunk 3 above)
Comment (jonlld) [reply src/components/organisms/AlarmHistory.tsx:8]: As above

Comment (jonlld) [reply src/components/organisms/AlarmHistory.tsx]: See above

File: src/components/molecules/AlarmHistoryItem.tsx (hunk 1 above)
Comment (jonlld) [reply src/components/molecules/AlarmHistoryItem.tsx:6]: It's not being rendered but may be useful to check only true items are being passed in - we can strip it out if you feel that's cleaner just requires a little extra code in the parent component

Comment (jonlld) [reply src/components/molecules/AlarmHistoryItem.tsx]: Have removed as we will move the filtering logic to backend

**Review by Takashicc — APPROVED** [review]

**Comment by Takashicc** [comment]
Waiting for backend side fix finishes...

**Pull request merged**